content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -81,7 +81,10 @@ Make sure you have vault running in a docker container with kubernetes auth enab
 ```bash
 docker run --name jx-k3s-vault -d --cap-add=IPC_LOCK -e 'VAULT_DEV_ROOT_TOKEN_ID=myroot' -e 'VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200' --net host vault:latest
 ```
-
```

Comment:
Perhaps it would help to add one line about what WSL2 Is?

Reply:
Yeah sure, I would add a link to official WSL2 docs as well.